.gitignore

```diff
@@ -0,0 +1 @@
+venv/
```

Comment:
Can you please add a newline here? 

Also, great idea to include the virtualenv binaries in gitignore. If everyone uses venv then this will automatically help.

Reply:
Newline added in latest commit.

Reply:
👍